base/uk.ac.stfc.isis.ibex.opis/resources/opi_info.xml

```diff
@@ -2653,5 +2653,14 @@
         </macros>
       </value>
     </entry>
+    <entry>
+      <key>WISH Vacuum PLC</key>
+      <value>
+        <type>UNKNOWN</type>
+        <path>wishvacuumplc.opi</path>
```

Comment:
Can we change the opi file name to wish_vacuum_plc? I think we do this for other file names with multiple words too